src/space.h

```diff
@@ -184,7 +187,7 @@ typedef struct JumpPoint_ {
    double sina; /**< Sinus of the angle. */
    int sx; /**< X sprite to use. */
    int sy; /**< Y sprite to use. */
-} JumpPoint;
+};
```

Comment:
Why did you remove the usual `typedef struct Foo_ { ... } Foo;` idiom? It's the usual way to do these.

🕵️

Reply:
It needs to be forward declared in order to use it in the struct definition